update_version.sh

```diff
@@ -1,31 +1,49 @@
 #!/usr/bin/env bash
 
-if [[ ("$2" == "poetry") || (("$2" == "") && ( -f pyproject.toml)) ]]; then
+function is_output_valid() {
+    line_numbers=$(eval "echo $1 | wc -l")
+    is_version_correct=$(eval "echo $1 | grep -E '[0-9.]?'")
```

Comment:
I'm not sure that `0-9.]?` is enough.
As I said, if git something will fail, the version can contain some error message which is easily matched by this `0-9.]?` regexp. I suggest  `^([0-9]+\.)+[0-9]+$` which is strict and covers all our existing versions.
In case it will not cover some versions, we can update it later according the needs.